aws/resource_aws_glue_trigger.go

```diff
@@ -126,6 +126,11 @@ func resourceAwsGlueTrigger() *schema.Resource {
 					glue.TriggerTypeScheduled,
 				}, false),
 			},
+			"workflow_name": {
```

Comment:
Can you please add documentation for this new argument? Thanks! `website/docs/r/glue_trigger.html.markdown`